UKHomeOffice/sgmr-service
# SPCR-263 Add onboarding component to dashboard

### AC / Description of changes
- refactor Dashboard to v2
- move setting userContext to happen on signin (in preparation for removal as it's not needed in v2)
- remove userContext from welcome banner
- add onboarding component
- add page level error messages



### To Test
- delete your account
- register your account again
- validate your account and login
- you should have 0 people, 0 pleasure crafts, 0 voyages
- > you should see the onboarding component on the dashboard
- add a person
- return to dashboard
- > you should not see the onboarding component

- delete your account
- register your account again
- validate your account and login
- you should have 0 people, 0 pleasure crafts, 0 voyages
- > you should see the onboarding component on the dashboard
- add a pleasure craft
- return to dashboard
- > you show not see the onboarding component

- delete your account
- register your account again
- validate your account and login
- you should have 0 people, 0 pleasure crafts, 0 voyages
- > you should see the onboarding component on the dashboard
- add a voyage
- return to dashboard
- > you show not see the onboarding component


### Notes
- the onboarding flow only contains the pleasure craft form at this point

---
* remember to merge to feature branches not master *
